public/framework/catalog_cache/cache_manager.ts

```diff
@@ -75,15 +76,56 @@ export class CatalogCacheManager {
     } else {
       const defaultCacheObject = {
         version: CATALOG_CACHE_VERSION,
-        accelerations: [],
-        lastUpdated: '',
-        status: CachedDataSourceStatus.Empty,
+        dataSources: [],
```

Comment:
This is because the `dataSources: []` contains the above? 
```
        accelerations: [],
        lastUpdated: '',
        status: CachedDataSourceStatus.Empty,
```

Reply:
yes!